db/migrate/20180215112954_fix_whitehall_remove_change_note.rb

```diff
@@ -0,0 +1,27 @@
+class FixWhitehallRemoveChangeNote < ActiveRecord::Migration[5.1]
+  disable_ddl_transaction!
+  
+  def up
+    # Find change notes
+    d = Document.where(content_id: "b3b96cb9-e7fa-493c-87a5-31d71d79c7a7").first
+    if d.present?
+      change_notes = d.editions.map(&:change_note).compact
+      change_notes.select{ |change_note| change_note.note == "Added funding allocations for individual LAs for 2018 to 2019."}
+  
+      # Get rid of change notes
+      change_notes.map(&:destroy)
```

Comment:
I wonder if this might delete all the change notes because the `select` call above doesn't get reassigned to the `change_notes` variable?

Reply:
Thanks! Corrected here: https://github.com/alphagov/publishing-api/pull/1166